toto/runlib.py

```diff
@@ -27,11 +27,18 @@
 import sys
 import os
 import tempfile
+import logging
+
 
 # POSIX users (Linux, BSD, etc.) are strongly encouraged to
 # install and use the much more recent subprocess32
 if os.name == 'posix' and sys.version_info[0] < 3:
-  import subprocess32 as subprocess
+  try:
+    import subprocess32 as subprocess
+  except Exception, e:
```

Comment:
Good idea. Will do in https://github.com/in-toto/in-toto/issues/6#issuecomment-257575261.